scripts/train/train.py

```diff
@@ -527,6 +527,20 @@ def main(cfg: DictConfig) -> Trainer:
         compile_config=compile_config,
     )
 
+    # Optionally just save an HF checkpoint
+    if train_cfg.only_hf_checkpoint:
+        hf_checkpointer_callbacks = [
+            c for c in callbacks if isinstance(c, HuggingFaceCheckpointer)
+        ]
+        if len(hf_checkpointer_callbacks) == 0:
+            raise ValueError(
+                'No HuggingFaceCheckpointer callback found, but only_hf_checkpoint was set to True. Please add a HuggingFaceCheckpointer.',
+            )
+
```

Comment:
should raise error or warning if > 1 hf checkpointer

Reply:
added warning